AbstractSQLRules2SQL.ometajs

```diff
@@ -743,6 +744,23 @@ export ometa AbstractSQLRules2SQL {
 			)
 		),
 
+	Case :indent =
+		[
+			'Case'
+			[	When(indent)+:whens
+			]
+			AnyValue(indent):val
```

Comment:
The else value is optional in postgres, so it's probably better to have this as `AnyValue(indent)?:val` and then something like `val ? ' ELSE ' + val : ''`

Reply:
As above, it should be clear and optional.